README.md

```diff
@@ -108,42 +109,25 @@ define(['form'], function(form) {
 ``` txt
  required: This field is required
  min: Please enter a value greater than or equal to {0}
+ max: 'Please enter a value greater than or equal to {0}'
  maxlength: Please enter a value with max length less than or equal to {0}
  pattern: Please enter a valid value
  email: Please enter a valid email address
 ```
 
-#### appendMessage
-`.append(selector, text)`
+#### validateField
+`.validateField(selectorOrElements)`
 
 ###### Description:
-Append validation messages
+Validate individual fields
 
 ###### Parameters:
-> selector: String 
-
-> text: String
-
-###### Sample:
-``` js
-define(['form'], function(form) {
-  form.append('label[for="date"]', 'dd/mm/yyyy'); //This will append <span class="message">dd/mm/yyyy</span>. Note that this element will be removed when the user starts to type another value.
-});
-```
-
-#### removeValidationErrors
-`.removeValidationErrors(selector)`
-
-###### Description:
-Removes all validation messages from selected form
-
-###### Parameters:
-> selector: String 
+> selectorOrDocElement: doc-amd object or String //A CSS selector. Note that, if it is a class name with dots, the dots must be escaped. E.g.: doc(".my\\\\.class")
 
 ###### Sample:
 ``` js
 define(['form'], function(form) {
-  form.removeValidationErrors('#form'); //This will remove all validation messages appended
+  form.validateField($('input[name=example1]')); //This will validate the field with name 'example1' and it will show a error message.
```

Comment:
Could create more examples?

@tisvasconcelos

Reply:
What would be?

@tcelestino